docs/tasks/access-application-cluster/create-external-load-balancer.md

```diff
@@ -120,11 +136,24 @@ This feature can be activated by adding the beta annotation below to the metadat
         "selector": {
           "app": "example"
         },
-        "type": "LoadBalancer"
+        "type": "LoadBalancer",
+        "externalTrafficPolicy": "Local"
       }
     }
 ```
 
+Below you could find the deprecated beta annotations. We may stop supporting them after
+v1.7. Please consider using API fields instead.
+
+* `service.beta.kubernetes.io/external-traffic` annotation <-> `Service.Spec.ExternalTrafficPolicy` field
+* `service.beta.kubernetes.io/healthcheck-nodeport` annotation <-> `Service.Spec.HealthCheckNodePort` field
+
+`service.beta.kubernetes.io/external-traffic` annotation has a different set of values
+comparing to `Service.Spec.ExternalTrafficPolicy` field, the values match as follows:
```

Comment:
suggested edit: "...compared to the \`Service.Spec.ExternalTrafficPolicy\` field. The values match as follows:"